zarr/v3/metadata.py

```diff
@@ -191,9 +199,13 @@ def dtype(self) -> np.dtype:
     def ndim(self) -> int:
         return len(self.shape)
 
-    def get_core_metadata(self, runtime_configuration: RuntimeConfiguration) -> CoreArrayMetadata:
-        return CoreArrayMetadata(
-            shape=self.shape,
+    def get_chunk_metadata(
+        self, _chunk_coords: ChunkCoords, runtime_configuration: RuntimeConfiguration
```

Comment:
Neither `_chunk_coords` nor `runtime_configuration` get used here. I'm guessing that this function takes `_chunk_coords` as an argument in anticipation of irregular chunking, but what's the use for `runtime_configuration`? Maybe a docstring could help here

Reply:
> I'm guessing that this function takes _chunk_coords as an argument in anticipation of irregular chunking,

Exactly! We can leave it out for now.